assets/sass/utilities/_overflow.scss

```diff
@@ -0,0 +1,24 @@
+/**
+ * Overflow utility styles.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+
+	.googlesitekit-overflow-auto {
+		overflow: auto;
+	}
```

Comment:
Is this because of the textarea or why?

Reply:
I believe this is to allow the `pre` element to scroll horizontally when the content is wider than the element itself, so that it doesn't potentially stretch larger than its container. Can you confirm @eugene-manuilov ?

Reply:
Yes, that's correct.